packages/trader/src/Stores/Modules/Trading/trade-store.js

```diff
@@ -294,7 +297,7 @@ export default class TradeStore extends BaseStore {
             clientInitListener: action.bound,
             networkStatusChangeListener: action.bound,
             themeChangeListener: action.bound,
-            turbos_barrier_choices: observable,
+            turbos_barrier_choices: observable.struct,
```

Comment:
If we don't add `.struct`, components which receive `turbos_barrier_choices` as props (per `connect`) will be rerendered every time when we get WS answer (even if we cover  `turbos_barrier_choices` with `React.useMemo`)